README.md

```diff
@@ -1,11 +1,15 @@
 # Imgproxy
+
 [![Build Status](https://github.com/bmuller/imgproxy/actions/workflows/ci.yml/badge.svg)](https://github.com/bmuller/imgproxy/actions/workflows/ci.yml)
-[![Hex pm](http://img.shields.io/hexpm/v/imgproxy.svg?style=flat)](https://hex.pm/packages/imgproxy)
-[![API Docs](https://img.shields.io/badge/api-docs-lightgreen.svg?style=flat)](https://hexdocs.pm/imgproxy/)
+[![Module Version](https://img.shields.io/hexpm/v/imgproxy.svg)](https://hex.pm/packages/imgproxy)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/imgproxy/)
+[![Total Download](https://img.shields.io/hexpm/dt/imgproxy.svg)](https://hex.pm/packages/imgproxy)
+[![License](https://img.shields.io/hexpm/l/imgproxy.svg)](https://github.com/bmuller/imgproxy/blob/master/LICENSE.md)
+[![Last Updated](https://img.shields.io/github/last-commit/bmuller/imgproxy.svg)](https://github.com/bmuller/imgproxy/commits/master)
```

Comment:
I'm not sure these "badges" are all that informative.  Stylistically, I'd like to omit.
```suggestion
```

Reply:
Noted. Removed.

Reply:
Noted. Removed.